registry.json

```diff
@@ -3014,5 +3014,20 @@
     "telegram": "alice_ops",
     "operatorName": "Nik Marsh",
     "github": "Skilov3"
-  }
+ },
+   {
+    "ticker": "GOALS",
+    "poolId": "pool1nh9j5qucsddxkzn3m4dm4gxkhwtpya4wm4e0ye2u0puaqgkcm65",
```

Comment:
I think your pool id is actually 9dcb2a0398835a6b0a71dd5bbaa0d6bb961276aedd72f2655c7879d0, please can you correct?